master/pkg/model/user.go

```diff
@@ -21,6 +21,7 @@ type User struct {
 	ID           UserID      `db:"id" json:"id"`
 	Username     string      `db:"username" json:"username"`
 	PasswordHash null.String `db:"password_hash" json:"-"`
+	DisplayName  null.String `db:"display_name" json:"-"`
```

Comment:
password_hash has the `json:"-"` thing just to make sure we don't share it. This should have display_name